sos/plugins/openstack_aodh.py

```diff
@@ -38,11 +38,17 @@ def setup(self):
 
         self.limit = self.get_option("log_size")
         if self.get_option("all_logs"):
-            self.add_copy_spec("/var/log/aodh/",
-                               sizelimit=self.limit)
+            self.add_copy_spec([
+                "/var/log/aodh/",
+                "/var/log/httpd/aodh*"],
+                sizelimit=self.limit
+            )
```

Comment:
Please fix this to place the closing `)` on the same line as the final keyword argument. This is the accepted style in `sos`.

Reply:
This should be written as:

```python
    self.add_copy_spec([
        "/var/log/aodh/",
        "/var/log/httpd/aodh*"
    ], sizelimit=self.limit)
```